### ISSUE TITLE
Breakpoints do not trigger for scripts ran as Administrator in windows 11

### ISSUE THREAD

**Issue by jtaylor3rd** [zero-plusplus/vscode-autohotkey-debug#297]
with the following vscode launch.json configuration:

```json
        {
            "type": "ahk",
            "request": "launch",
            "name": "AutoHotkey Debugger",
            "program": "${file}",
            "stopOnEntry": false,
            "runtime": "C:\\Program Files\\AutoHotkey\\v1.1.36.02\\AutoHotkeyU64.exe"
        },
        {
            "name": "Attach to AutoHotkey",
            "type": "autohotkey",
            "request": "attach",
            "port": 9000,
            "runtime": "C:\\Program Files\\AutoHotkey\\v1.1.36.02\\AutoHotkeyU64.exe",
            "useDebugDirective": true,
            "useOutputDebug": true
        },
```

I debug main.ahk using the "launch" configuration. main.ahk then starts another script with the following code:

```
Run, *RunAs %fullScriptPath%,, UseErrorLevel, pId ; fullScriptPath = someScript.ahk
```

I then use the "attach" configuration to debug someScript.ahk. vscode IDE says the attachment is successful, but any breakpoints I set in someScript.ahk is not triggered by the debugger. If I run someScript.ahk without administrator privileges, breakpoints trigger as expected.

How can I get the "attach" configuration to trigger breakpoints for scripts ran as administrator?

Love the extension, thanks!

**Comment by zero-plusplus** [comment]
Thank you for your report. This is a very valuable and important report as I never debug scripts that require administrator rights.

The attach to script is achieved by running an [AutoHotkey script](https://github.com/zero-plusplus/vscode-autohotkey-debug/blob/d00790d5eb8734069f39ec2edd7fab213d7d81fb/src/util/AutoHotkeyLuncher.ts#L91-L107) that sends a message to initiate it.
I guess that the your script executed with administrator rights rejects this message.

In other words, using administrative privileges to send attach messages may solve this problem.

Specifically, try running vscode with administrator rights.

**Comment by jtaylor3rd** [comment]
> Thank you for your report. This is a very valuable and important report as I never debug scripts that require administrator rights.
> 
> The attach to script is achieved by running an [AutoHotkey script](https://github.com/zero-plusplus/vscode-autohotkey-debug/blob/d00790d5eb8734069f39ec2edd7fab213d7d81fb/src/util/AutoHotkeyLuncher.ts#L91-L107) that sends a message to initiate it. I guess that the your script executed with administrator rights rejects this message.
> 
> In other words, using administrative privileges to send attach messages may solve this problem.
> 
> Specifically, try running vscode with administrator rights.

Yep, that worked! Appreciate the fast reply and thank you.
